docs/service-mesh/05-01-custom-configuration.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Istio custom configuration
+type: Configuration
+---
+Istio installation in Kyma uses the [IstioOperator](https://istio.io/docs/reference/config/istio.operator.v1alpha1/) API.
+Kyma provides the default IstioOperator configurations for local (Minikube) and cluster installations.
+
+You can add a custom Istio Operator definition that overrides the default settings.
+
+The definition you provide may be a partial one (you don't have to specify all options). In that case it will be merged with the defaults.
```

Comment:
```suggestion
The definition you provide may be a partial one with not all the options specified. In that case, it will be merged with the defaults.
```